backend/application/syson-sysml-export/src/main/java/org/eclipse/syson/sysml/export/SysMLElementSerializer.java

```diff
@@ -127,15 +133,22 @@ public class SysMLElementSerializer extends SysmlSwitch<String> {
      * @param indentation
      *            the string used to indent the file
      */
-    public SysMLElementSerializer(String lineSeparator, String indentation, NameDeresolver nameDeresolver) {
+    public SysMLElementSerializer(String lineSeparator, String indentation, NameDeresolver nameDeresolver, Consumer<Status> reportConsumer) {
         super();
         this.lineSeparator = lineSeparator;
         this.indentation = indentation;
         this.nameDeresolver = nameDeresolver;
+        if (reportConsumer == null) {
+            this.reportConsumer = r -> {
+            };
+        } else {
+            this.reportConsumer = reportConsumer;
+
```

Comment:
please do not let empty lines like this one, here and everywhere else

Reply:
Done